### PR TITLE
Nutanix:virt-who config UI cases support for SCA enabled

### PR DESCRIPTION
**Pull request by yanpliu**
Cases PASS:
```

(robottelo_vv) [virtwho@dell-per740-68-vm-04 robottelo]$ pytest  ./tests/foreman/virtwho/ui/test_nutanix_sca.py  --disable-pytest-warnings -q
......                                                                                                                                                                                                      [100%]
6 passed, 6 deselected, 11 warnings in 1408.20s (0:23:28)

```

### CODE REVIEW

File: tests/foreman/virtwho/conftest.py [hunk 1]
```diff
@@ -52,3 +52,51 @@ def test_foo(session):
...
+@pytest.fixture(scope='module')
+def module_user_sca(request, module_target_sat, module_org, module_location):
+    """Creates admin user with default org set to module org and shares that
+    user for all tests in the same test module. User's login contains test
+    module name as a prefix.
+
+    :rtype: :class:`nailgun.entities.Organization`
+    """
+    # take only "module" from "tests.foreman.virtwho.test_module"
+    test_module_name = request.module.__name__.split('.')[-1].split('_', 1)[-1]
```
Comment (JacobCallahan) [inline tests/foreman/virtwho/conftest.py:66]: do you know of cases where this wouldn't work?
```suggestion
    test_module_name = request.module.__name__.split('_', 1)[-1]
```

**Comment by yanpliu** [comment]
Could you please help to review and merge this PR? because some other UI PRs depend on this PRs change. Many thanks.

File: tests/foreman/virtwho/ui/test_nutanix_sca.py [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+    def test_positive_hypervisor_id_option(
+        self, module_sca_manifest_org, virtwho_config, session_sca, form_data
+    ):
+        """Verify Hypervisor ID dropdown options.
+
+        :id: 8f5771bd-4b74-49a7-93bb-31eb8e467477
+
+        :expectedresults:
+            1. hypervisor_id can be changed in virt-who-config-{}.conf if the
+            2. dropdown option is selected to uuid/hwuuid/hostname.
```
Comment (adarshdubey-star) [inline tests/foreman/virtwho/ui/test_nutanix_sca.py:107]:
```suggestion
            1. hypervisor_id can be changed in virt-who-config-{}.conf if the
                dropdown option is selected to uuid/hwuuid/hostname.
```

File: tests/foreman/virtwho/ui/test_nutanix_sca.py [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+    def test_positive_prism_central_prism_flavor_option(
+        self, module_sca_manifest_org, virtwho_config, session_sca, form_data
+    ):
+        """Verify prism_flavor dropdown options.
+
+        :id: 621abef9-3629-41ee-9977-0089a558e79b
+
+        :expectedresults:
+            1. prism_flavor can be changed in virt-who-config-{}.conf if the
+            2. dropdown option is selected to prism central.
```
Comment (adarshdubey-star) [inline tests/foreman/virtwho/ui/test_nutanix_sca.py:191]:
```suggestion
            1. prism_flavor can be changed in virt-who-config-{}.conf if the
                dropdown option is selected to prism central.
```

**Review by adarshdubey-star — APPROVED** [review]
Ack, pending small suggestions

**Pull request merged**